connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectStreamHandler.scala

```diff
@@ -47,34 +48,49 @@ class SparkConnectStreamHandler(responseObserver: StreamObserver[ExecutePlanResp
     extends Logging {
 
   def handle(v: ExecutePlanRequest): Unit = SparkConnectArtifactManager.withArtifactClassLoader {
-    val session =
-      SparkConnectService
-        .getOrCreateIsolatedSession(v.getUserContext.getUserId, v.getSessionId)
-        .session
+    val sessionHolder = SparkConnectService
+      .getOrCreateIsolatedSession(v.getUserContext.getUserId, v.getSessionId)
+    val session = sessionHolder.session
+
     session.withActive {
+      val debugString = try {
+        Utils.redact(
+          session.sessionState.conf.stringRedactionPattern,
+          ProtoUtils.abbreviate(v).toString)
+      } catch {
+        case NonFatal(e) =>
+          logWarning("Fail to extract debug information", e)
+          "UNKNOWN"
+      }
+
+      val executeHolder = sessionHolder.createExecutePlanHolder(v)
+      session.sparkContext.setJobGroup(
+        executeHolder.jobGroupId,
+        s"Spark Connect - ${StringUtils.abbreviate(debugString, 128)}",
```

Comment:
I am a bit annoyed by this line in the console output:
`Spark Connect - ${StringUtils.abbreviate(debugString, 128)`
Is it enough to only have session id etc here? As this code produce a multi-line json output, it does not contain any useful info except than session id, but adds a lot of line of [ { etc.

Reply:
In the Spark UI it looks ok. Can we decouple this from this PR if you want an improvement?

Reply:
Certainly, not related to this PR at all.